### PR TITLE
LongShortSwitch - 'Toggle' - toggling but tests failing

### PR DESCRIPTION
**Pull request by Alastair2D**

### CODE REVIEW

**Comment by hjpugh** [comment]
Happy with what's going on here, have worked on the majority of it. We still need to try and find out why there is an issues with the toggle-switch - having tried a large number of approaches, the cause of this is still unfortunately alluding us.... and the error message is essentially useless.

File: components/LongShortSwitch.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    };
+  }
+
+  // componentDidUpdate() {
+  //   return false;
+  // }
+
+  flickSwitch = (status) => {
+    if (status === true) {
+      this.setState({
```
Comment (andrewwood2) [inline components/LongShortSwitch.js]: Looks like both state variables do the same thing. Probably can consolidate.

File: redux/actions/sites.js [hunk 2]
```diff
@@ -3,6 +3,11 @@ export function nextInjSite() {
     type: 'sites-next-injection-site'
   };
 }
+export function prevInjSite() {
```
Comment (andrewwood2) [inline redux/actions/sites.js]: Looks like this can be deleted

File: screens/ConfirmModal.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import { connect } from 'react-redux';
+import { saveInj } from '../redux/actions/history';
+import { nextInjSite, rotateNSites } from '../redux/actions/sites';
+
+export class ConfirmModal extends Component {
+  state = {
+    modalVisible: false,
+  };
+
+  setModalVisible(visible) {
```
Comment (andrewwood2) [inline screens/ConfirmModal.js]: Rename argument to visibility

**Pull request merged**